assembl/static2/js/app/components/harvesting/harvestingBox.jsx

```diff
@@ -82,10 +83,13 @@ class DumbHarvestingBox extends React.Component<Props, State> {
     super(props);
     const { extracts, cancelHarvesting } = this.props;
     const hasExtract = extracts ? extracts.length > 0 : false;
-    const extract = this.getCurrentExtract(0);
+    const { hash } = window.location;
+    const hashExtractId = hash !== '' ? hash.split('#')[2] : null;
```

Comment:
hashExtractId can be undefined here if you don't have a second '#" in the url.

Reply:
I handle the case when hashExtractId is undefined after